src/mscorlib/src/System/Diagnostics/Stackframe.cs

```diff
@@ -255,13 +255,13 @@ public override String ToString()
                 {
                     Type[] typars = ((MethodInfo)method).GetGenericArguments();
 
-                    sb.Append("<");
+                    sb.Append('<');
```

Comment:
Right. The interning is per process for JIT and per module for NGen - it is one-time allocation saved per character per module in the NGen case.